src/StackTask/Program.cs

```diff
@@ -1,2 +1,78 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using System;
```

Comment:
I see bin and obj folder are pushed to the repo, we should ignore those. Please lets remove them and add a `.gitignore` file.

Reply:
Done